app/(main)/settings/team/billing-section.tsx

```diff
@@ -36,7 +37,7 @@ export default async function BillingSection() {
 						>
 							<BillingButton
 								subscriptionId={team.activeSubscriptionId}
-								teamDbId={team.dbId}
+								team={team}
```

Comment:
:memo: `BillingSection` and `BillingButton` are both React Server Components and therefore not part of the bundle, but changed to eliminate teamDbId as the props interface.

The reason for passing team instead of teamId is that team.dbId needs to be passed to the stripe meta data to which it is passed.

This time, since it is an RSC, we passed it as team for simplicity, but if `BillingButton` is a client component, it is appropriate to pass `teamId` and fetch `teamDbId` with `teamId` in the backend.